### DISCUSSION TITLE
2008/04/14/%E9%81%87%E5%88%B0%E7%83%AD%E6%83%85%E7%9A%84%E7%BE%8E%E5%9B%BD%E6%AF%8D%E5%A5%B3%E4%BA%8C%E4%BA%BA

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#214] [Announcements]
# 2008/04/14/%E9%81%87%E5%88%B0%E7%83%AD%E6%83%85%E7%9A%84%E7%BE%8E%E5%9B%BD%E6%AF%8D%E5%A5%B3%E4%BA%8C%E4%BA%BA

刚才和几个同事去一家叫 Salt Lick 的店吃烤肉。我们几个研究菜单时候的迷惑表情被旁边一位阿姨看到了，她和她女儿刚吃好饭正准备走。于是主动的跑上来给我们介绍这家店的特色菜，顺带又开始 介绍德州风情。而她的女儿更是现身说法，指着身上的衣服，给我们介绍了一下德州特色服饰。女儿大约20出头，绝对是难得一见的大美女，身材、皮肤都非常棒；而母亲看上去也相当年轻，完全不像一般美国人老的那么比较快。不是她自己说他们是母女，我以为她才三十多呢。

https://blog.qizhen.xyz/2008/04/14/%E9%81%87%E5%88%B0%E7%83%AD%E6%83%85%E7%9A%84%E7%BE%8E%E5%9B%BD%E6%AF%8D%E5%A5%B3%E4%BA%8C%E4%BA%BA

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋
2008年04月22日 12:48 编辑
哈哈哈~~~~~

回复
破
2008年04月21日 16:44 编辑
上次去US开会，半夜三更和同事开车出去，后来有点小事在路边停了一下，开了双跳灯，结果不一会一个路过的美女特地停车来问我们是否需要帮忙，可以帮我们修车云云，我们当时感动坏了，心想这老美可真是民风淳朴啊~

回复
sandy
2008年04月17日 09:02 编辑
通过我在美国一年半的所见，知道最后你的陈述是实在的，但是Jenna说的也很对。不要拿我们弱点跟人家的优点比。他们的缺点也是多了去了。不过，我从来都是很自豪的告诉老美们我是中国人，不管他们对中国的看法是什么。

回复
奕瑋
2008年04月16日 22:13 编辑
主席所言极是，严重支持。

回复
xiaoji
2008年04月15日 15:55 编辑
在我旅游的经历中，我也有这样的感觉。然后总会和国内做比较，觉得有点痛心。
也许，我们应该这样想，我们只是拿别人好的一面和我们不足的一面做了比较；别人也会拿我们好的一面和他们不足的做对比。还是那句话，要相信中国会越来越强大，国人的素质会越来越好。到时候，我们每个人出去可以很自豪的说：I come from China, Mainland china.

回复
Gladys Pang
2008年04月14日 21:43 编辑
我也觉得德州女人是比较好看。很多人是西班牙血统，还有混血的。

回复
永青
2008年04月14日 14:42 编辑
我不得不说，阮大侠怎么总是有这样的经历。。。

回复
duyunzhu
2008年04月14日 14:32 编辑
我承认我是被 “绝对是难得一见的大美女，身材、皮肤都非常” 吸引进来的

回复
Xiaohong
2008年04月14日 10:23 编辑
没有PP，可惜啊，可惜

回复
